atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -199,21 +209,49 @@ private SweepResults doRun(TableReference tableRef,
     }
 
     /**
-     * Returns the number of deleted (cell, timestamp) pairs.
+     * Returns the number of blocks that were deleted.
```

Comment:
This is a fair question. I'm trying to introduce as much as the 'block' nomenclature as possible, in order to get people used to it, since that's the term we use when referring to it. I've clarified this comment.